kubetest/e2e.go

```diff
@@ -393,6 +436,8 @@ type deployer interface {
 	IsUp() error
 	SetupKubecfg() error
 	Down() error
+	FederationUp() error
```

Comment:
This is a duplicate of the Up() method. Will you refactor things such that there is a --deployment=federation option?